### ISSUE TITLE
We should switch all docs and process to yarn from npm

### ISSUE THREAD

**Issue by joncodo** [merico-dev/lake#183]
Yarn is the open source standard and has many advantages to npm. Nothing changes locally for anyone. You just replace all `npm` commands with the word `yarn`

IE: 

`npm i` -> `yarn i`
`npm run test` -> `yarn run test`

> One of the main difference between NPM and Yarn is how they handle the package installation process. Yarn installs packages in parallel. Yarn is optimized to fetch and install multiple packages simultaneously. If you are installing five packages, and two are taking a long time to install, Yarn will go over and install the packages side by side.

> Yarn checks behind the scenes and make sure that you’re not downloading rogue scripts or stuff that can conflict with your project dependencies. Security is one of Yarn’s core features.

> There are not many comparisons to be drawn between the two. You can use Yarn pretty much in every case that you would NPM. It is meant to be a drop-in replacement.

You can read more here: https://www.section.io/engineering-education/npm-vs-yarn-which-one-to-choose/

**Comment by joncodo** [comment]
Maybe we don't need to do this but I just thought I would suggest it :)

**Comment by joncodo** [comment]
We can close this for sure. No worries. Can you please tell me why it is on hold or would you like to close it? Feel free to re-open if you still want this.
